src/Commands/AdminCommands/CleanupCommand.php

```diff
@@ -327,7 +351,7 @@ private function getQueries($settings)
     public function execute()
     {
         $message = $this->getMessage();
-        $chat_id = $message->getFrom()->getId();
+        $chat_id = $message->getChat()->getId();
```

Comment:
Yes, I know that, which is why I thought you put that there in the first place :smile:
Because with `getChat`, it posts any error or the final summary visible for anyone in a chat to read.
IMHO, maintenance stuff should be managed privately.